### DISCUSSION TITLE
Question about password hashing and salting

### DISCUSSION THREAD

**Discussion by seras42** [davidmartos96/sqflite_sqlcipher#87] [Q&A] [answered]
I appreciate and thank the developers for making this package.

I wanted to ask how does the password encryption work exactly, if i pass the string "password" is that then hashed and salted with unique salt when used to make the database? Or simply the string is hashed and then used for creating the encrypted database? The main reason i am asking this is because i was unable to find any documentation.

**Comment by davidmartos96** [comment, marked as answer]
@seras42 Hello, this is an unofficial Flutter package, which wraps the official Android and iOS client libraries that the team behind SQLCipher develops.
We won't be able to answer you unfortunately. I'd recommend reaching out to the SQLCipher team.
They have a forum for questions https://discuss.zetetic.net/

**Reply by seras42** [reply]
Thanks for the quick response. That is all i wanted to know. I was wondering if this package comes from that.
